blaze-server/src/main/scala/org/http4s/server/blaze/BlazeBuilder.scala

```diff
@@ -238,7 +241,7 @@ class BlazeBuilder[F[_]](
 
 @deprecated("Use BlazeServerBuilder instead", "0.20.0-RC1")
 object BlazeBuilder {
-  def apply[F[_]](implicit F: ConcurrentEffect[F], timer: Timer[F]): BlazeBuilder[F] =
+  def apply[F[_]](D: Dispatcher[F])(implicit F: Async[F]): BlazeBuilder[F] =
```

Comment:
If we have to change the signature, I don't think there's any reason to keep the deprecation.  This is only there to help people for whom it no longer compiles.